src/xdocs/config_annotation.xml

```diff
@@ -142,14 +145,14 @@ public String getNameIfPresent() { ... }
         <p>
           Use the following configuration:
         </p>
-        <source>
-&lt;module name=&quot;AnnotationLocation&quot;&gt;
-  &lt;property name=&quot;allowSamelineMultipleAnnotations&quot; value=&quot;true&quot;/&gt;
-  &lt;property name=&quot;allowSamelineSingleParameterlessAnnotation&quot;
-    value=&quot;false&quot;/&gt;
-  &lt;property name=&quot;allowSamelineParameterizedAnnotation&quot; value=&quot;false&quot;/&gt;
-&lt;/module&gt;
-        </source>
+        <source><![CDATA[
```

Comment:
This change doesn't seem necessary. Nothing was changed besides adding the CDDATA.
Please revert.

Reply:
I don't remember, I probably just wanted to increase readability.
But as you don't like it, I reverted it.